setup/templates/steps/compose/02_services.html

```diff
@@ -64,6 +64,15 @@
   <i>Oletools scans documents in email attachements for malicious macros. It has a much lower memory footprint than a full-fledged anti-virus.</i>
 </div>
 
+<div class="form-check form-check-inline">
+  <label class="form-check-label">
+    <input class="form-check-input" type="checkbox" name="tika_enabled" value="true">
+    Enable Tika
+  </label>
+
+  <i>Tika scans documents in email attachments, process (OCR, keyword extraction) and then index them in a way they can be efficiently searched. This requires significant ressources (RAM, CPU and storage).</i>
```

Comment:
Please make more clear that you really need this for the ability to search through attachments.
It must also mention that currently only amd64 / x64 is supported.
e.g.
`Tika enables the functionality for searching through attachments. Tika scans documents in email attachments, process (OCR, keyword extraction) and then index them in a way they can be efficiently searched. This requires significant ressources (RAM, CPU and storage). This is currently not supported on ARM.
`

Reply:
That's 0a7797e9